ds3_java_cli/src/main/java/com/spectralogic/ds3cli/util/Utils.java

```diff
@@ -249,7 +248,16 @@ public static void restoreLastModified(final String filename, final com.spectral
             }
 
         } else {
-            LOG.warn("Object ("+ filename + ") does not contain a last modified field");
+            LOG.warn("Object (" + filename + ") does not contain a last modified field");
         }
     }
+
+    public static boolean isCollectionNullOrEmpty(ImmutableMap collection) {
```

Comment:
There are `Guard` methods on the sdk that should already implement this functionality.  See [Guard](https://github.com/SpectraLogic/ds3_java_sdk/blob/master/ds3-sdk/src/main/java/com/spectralogic/ds3client/utils/Guard.java)